chatmeleon-server/package.json

```diff
@@ -3,21 +3,27 @@
   "version": "1.0.0",
   "description": "",
   "main": "src/index.ts",
-  "type": "module",
   "scripts": {
-    "start": "node --loader ts-node/esm src/index.ts"
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "dev": "nodemon"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@types/node": "^20.8.0",
+    "@prisma/client": "^5.4.2",
     "dotenv": "^16.3.1",
```

Comment:
Why are these added in this PR? It would cause conflict after the fix from #5 is merged. I think it's not needed to fix it here but you can keep it and fix the conflict later if you want.

Reply:
I didn't fix anything ehe

Reply:
All good sir